airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -639,6 +639,7 @@ def __init__(
         request_id: str | None = None,
         delete_on_error: bool = True,
         use_if_exists: bool = True,
+        retry_if_resource_not_ready: int | None = None,
```

Comment:
Doesn't it sound more like a bool parameter, rather then int? I suggest `num_retries_if_resource_is_not_ready`, but  it's kind of a personal preference. 
Also, if we never check for None specifically, e.g. `is not None` shouldn't it be just `0` value by default?